sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -123,6 +124,24 @@ class DataFrameReader private[sql](sqlContext: SQLContext) extends Logging {
   }
 
   /**
+   * Loads input in as a [[DataFrame]], for data sources that support multiple paths.
+   * Only works if the source is a HadoopFsRelationProvider.
+   *
+   * @since 1.6.0
+   */
+  def load(paths: Seq[String]): DataFrame = {
```

Comment:
That makes sense.... Since it works for java too that eay
On Oct 17, 2015 7:14 AM, "Michael Armbrust" <notifications@github.com>
wrote:

> In sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala
> <https://github.com/apache/spark/pull/8416#discussion_r42274201>:
>
> > @@ -123,6 +124,24 @@ class DataFrameReader private[sql](sqlContext: SQLContext) extends Logging {
> >    }
> >
> >    /**
> > +   * Loads input in as a [[DataFrame]], for data sources that support multiple paths.
> > +   * Only works if the source is a HadoopFsRelationProvider.
> > +   *
> > +   * @since 1.6.0
> > +   */
> > +  def load(paths: Seq[String]): DataFrame = {
>
> This should be just a single load(paths: Array[String]) as in other parts
> of the API.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/8416/files#r42274201>.
>